lua/sol.lua

```diff
@@ -28,6 +28,8 @@ return function (net_conf, lcore)
 		error("Failed to allocate sol_conf")
 	end
 
+	staticlib.sol_assign_lcores(sol_conf, sol_lcores)
```

Comment:
I can't comment on the line above: `local sol_conf = staticlib.c.alloc_sol_conf(lcore)`. However, it must be updated as well as the Lua definition of `staticlib.c.alloc_sol_conf()`.